### PR TITLE
deprecated: replace getSchemaManager() by createSchemaManager()

### PR DESCRIPTION
**Pull request by victor-paumier**

### CODE REVIEW

File: sources/DBAL/SchemaMigrator.php [hunk 1]
```diff
@@ -17,7 +17,10 @@ public function __construct(Connection $connection)
     public function migrateSchema(): void
     {
         $platform = $this->connection->getDatabasePlatform();
-        $fromSchema = $this->connection->getSchemaManager()->createSchema();
+        $fromSchema = method_exists(Connection::class, 'createSchemaManager')
```
Comment (trompette) [inline sources/DBAL/SchemaMigrator.php:20]: I think it is `doctrine/dbal`'s job to provide an upgrade path for this change.

For example, when `doctrine/dbal` introduced `fetchFirstColumn()`, several versions supported the new method and that is why I could create a simple PR like https://github.com/trompette/php-feature-toggles/pull/22 to remove the deprecation.

**Review by trompette — COMMENTED** [review]
Hi again @victor-paumier,

I am pretty sure this pull request is a bad idea, see my justification below.

File: sources/DBAL/SchemaMigrator.php (hunk 1 above)
Comment (victor-paumier) [reply sources/DBAL/SchemaMigrator.php:20]: Well, I agree that you are right as it is also the case for other functions that are depreciated and have no equivalent solution today... Let's close this PR!

**Comment by trompette** [comment]
I have just created https://github.com/doctrine/dbal/issues/5198 to make a suggestion to `doctrine/dbal`.

**Pull request closed without merging**